### PR TITLE
docs: Fix ordering of items in the list

### PR DESCRIPTION
**Pull request by anithamuthu2022**
[DOC-104]

[DOC-104]: https://heartex.atlassian.net/browse/DOC-104?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/source/guide/tasks.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ short: Import data
 type: guide
 tier: all
 order: 120
-order_enterprise: 105
+order_enterprise:
```
Comment (makseq) [inline docs/source/guide/tasks.md:7]: is it ok to have this empty?

**Review by makseq — CHANGES_REQUESTED** [review]

File: docs/source/guide/tasks.md (hunk 1 above)
Comment (anithamuthu2022) [reply docs/source/guide/tasks.md]: @makseq https://github.com/heartexlabs/label-studio/pull/3442/files#diff-71883e6254217158bd24bf5a5629ff8eff44d0b4c54d8600a74216442200e32cR7

**Comment by anithamuthu2022** [comment]
@makseq this is ready for review/approval.

File: docs/source/guide/get_started.md [hunk 2]
```diff
@@ -216,16 +216,18 @@ Label Studio is available as a Community edition open source data labeling tool.
...
+  <li><a href="install.html">Install Label Studio</a>.</li>
+  <li><a href="start.html">Start Label Studio</a>.</li>
+  <li><a href="signup.html">Create accounts for Label Studio</a>.</li>
+  <li class="enterprise-only"><a href="manage_users.html">Restrict access to the project. Set up role-based access control. Only available in Label Studio Enterprise Edition</a>.</li>
+  <li><a href="setup_project.html">Set up the labeling project. Define the type of labeling to perform on the dataset and configure project setting </a>.</li>
+  <li><a href="setup.html">Set up the labeling interface. Add the labels that you want annotators to apply and customize the labeling interface</a>.</li>
+  <li><a href="tasks.html">Import data as labeling tasks</a>.</li>
+  <li><a href="labeling.html">Label and annotate the data</a>.</li>
+  <li><a href="quality.html">Review the annotated tasks. Only available in Label Studio Enterprise Edition</a>.</li>
+  <li class="enterprise-only"><a href="export.html">Export the labeled data or the annotations</a>.</li>
```
Comment (makseq) [inline docs/source/guide/get_started.md:229]: incorrect enterprise-only, it should be applied to item above - Review the annotated tasks.....

File: docs/source/guide/manage_users.md [hunk 3]
```diff
@@ -348,7 +348,7 @@ Each user has a personal Sandbox workspace that they can use to experiment with
 
 ### More menu to Pin or Unpin and Duplicate projects
 
-Use the more menu (three dots (**...**) located in the bottom-right of each project card) to explore the following features: 
+Use the more menu (three dots (**...**)) located in the bottom-right of each project card) to explore the following features:
```
Comment (makseq) [inline docs/source/guide/manage_users.md:351]: we don't need `)` here

**Review by makseq — CHANGES_REQUESTED** [review]

**Pull request closed without merging**